### ISSUE TITLE
Missing code artifacts (Lab 300)

### ISSUE THREAD

**Issue by AlessandroVol23** [awslabs/aws-well-architected-labs#287]
Hi,

I wanted to run some code like described in the lab, e.g. [here](https://wellarchitectedlabs.com/reliability/300_labs/300_testing_for_resiliency_of_ec2_rds_and_s3/6_failure_injection_az/)  run `python fail_az.py <vpc-id> <az>` but I cannot find the scripts. 

I see that the readme states 

> The labs cannot be run from GitHub. Please continue to use GitHub to log issues or make pull requests. To run the labs please use https://wellarchitectedlabs.com/

But I am not sure where I can find the code except for old commits like [here](https://github.com/awslabs/aws-well-architected-labs/blob/20577ff6891bdffffb18c5fb6e922a0a315eed6b/Reliability/300_Testing_for_Resiliency_of_EC2_RDS_and_S3/Code/FailureSimulations/python/fail_az.py), but I am pretty sure there is a reason why they were deleted.

Would be great if you could point me somewhere where I get that information :-) 

Basically my goal is to mock an AZ failure.

Thanks!

**Comment by AlessandroVol23** [comment]
Okay not sure but maybe it was removed due to the launch of FIS. https://aws.amazon.com/de/blogs/aws/aws-fault-injection-simulator-use-controlled-experiments-to-boost-resilience/

**Comment by setheliot** [comment]
No they were not removed.  We migrated to a different website presentation platform about a year ago

You can find the links to the scripts here https://wellarchitectedlabs.com/reliability/300_labs/300_testing_for_resiliency_of_ec2_rds_and_s3/2_configure_env/

If you just want to find the code on GitHub, it is here https://github.com/awslabs/aws-well-architected-labs/tree/master/static/Reliability/300_Testing_for_Resiliency_of_EC2_RDS_and_S3/Code/FailureSimulations
